### ISSUE TITLE
Allow create a textile component in Object

### ISSUE THREAD

**Issue by camcoq-start** [MTES-MCT/ecobalyse#1120]
### Problem

There is many specificities in Textile LCA simulator. 
Those specificities cannot be handled in component approach but we need to create textile component for chair or sofa or curtains.

### Solution

Get Textile LCA simulator compatible with component and so allow Alban create textile component for object.

Create the following processes (these are existing process but in front code): 
- [x] 3 spinning processes : 

- Nom= filature conventionnelle (40 Nm), Nom technique = filature conventionnelle (40 Nm), Source = Custom, unit= kg Electricité = 3,64Kwh  , chaleur= 0MJ , perte= 12% , densité= n/a, categories= manufacturing, material type = fibres organique, 
- Nom= filature non conventionnelle (40 Nm), Nom technique = filature non conventionnelle (40 Nm), Source = Custom, Kwh= 1,82 , MJ=0, loss= 12%, categories= manufacturing, material type = fibres organique 
- Nom= filage (40 Nm) , Nom technique = filage(40 Nm), Source = Custom, Kwh= 1,36 , MJ= 0 , loss= 3%, categories= manufacturing, material type = fibres synthétiques 

- [x] 1 weaving process:
- Nom = tissage ( 40Nm et 250 g/m2), Nom technique= tissage ( 40Nm et 250 g/m2), Source = Custom, unit = kg, chaleur = 0MJ , Electricité= 5,82Kwh* , material type = fibres organiques, fibres synthétiques, loss = 6,25

5,82*= 250(g/m2)*40 (Nm)/1,08/2*1000(g)/250(g/m2)*0,0003145 --> this is the function used in Textile LCA to calculate elec quantity. 

Note1 : for now, just one process is too limited to fully integrate Textile LCA simulator in component approach. It is enough for Object.

- [x] Add to categories for pre-treatment processes, the material type:
- bleaching : material type = fibres organiques
- degreasing : material type = fibres organiques
- washing : material type = fibres synthétiques

- [x] Add to the following raw materials textile processes, the material type:
- Production de PET, granulés : fibres synthétiques
- Production de fibre de coton: fibres organiques
- Production de fibre de coton bio: fibres organiques

Note3: For now we don't add the microfibre complement. This is mostly due to the using stage and this parameter lives in the material json file. (Material json file is excluded in component approach)
 
### Acceptance criteria

- [x] Create a component with raw and manufacturing textile processes in object simulator
- [ ] manufacturing process list depends on material type firstly chosen to create the element
- [x] loss are respected
- [x] component is located in asia with the default electrical and heat mix

### Test scenario
As admin, i am in back-office to create a "sofa textile"
- [ ] i duplicate an existing component in object
- [ ] i delete each element to create it from scratch
- [ ] i give as component title: "sofa textile"
- [x] i add 0,8kg of coton production to start my element
- [x] i add the spinning process by clicking on " add a manufacturing"
- [ ] i have a restricted list of manufacturing process (filter is material type = fibres organiques)
- [x] i add filature conventionnelle
- [x] i add tissage
- [x] i add blanchiement
- [x] i add dégraissage
- [x] i add 0,2kg of polyesther to start my second element
- [x] i add the spinning process by clicking on " add a manufacturing"
- [ ] i have a restricted list of manufacturing process (filter is material type = fibres synthétiques)
- [x] i add filage
- [x] i add tissage
- [x] final component mass is 1kg
- [ ] final ecoscore is 2402 pts - (200 + 164 pts due to microfibre complement)-(9+8+7 pts due to transport) = 2014 pts (according this reference and ignoring durability coef : https://ecobalyse.beta.gouv.fr/#/textile/simulator/ecs/eyJidXNpbmVzcyI6ImxhcmdlLWJ1c2luZXNzLXdpdGhvdXQtc2VydmljZXMiLCJjb3VudHJ5RHllaW5nIjoiUkFTIiwiY291bnRyeUZhYnJpYyI6IlJBUyIsImNvdW50cnlTcGlubmluZyI6IlJBUyIsImRpc2FibGVkU3RlcHMiOlsiZW5ub2JsaW5nIiwibWFraW5nIiwiZW9sIiwidXNlIiwiZGlzdHJpYnV0aW9uIl0sImZhYnJpY1Byb2Nlc3MiOiJ3ZWF2aW5nIiwibWFzcyI6MSwibWF0ZXJpYWxzIjpbeyJpZCI6ImVpLWNvdG9uIiwic2hhcmUiOjAuOH0seyJpZCI6ImVpLXBldCIsInNoYXJlIjowLjJ9XSwibnVtYmVyT2ZSZWZlcmVuY2VzIjoxMDAwMDAsInByaWNlIjoxMCwicHJvZHVjdCI6InRzaGlydCIsInN1cmZhY2VNYXNzIjoyNTAsInRyYWNlYWJpbGl0eSI6ZmFsc2UsInRyaW1zIjpbXSwidXBjeWNsZWQiOmZhbHNlfQ==

**Comment by camcoq-start** [comment]
A small difference between exemple and final result: 

<img width="869" alt="Image" src="https://github.com/user-attachments/assets/8d4661e7-04cc-4c5e-88d0-28877b96eb3f" />

Might be due to losses
